docs/index.asciidoc

```diff
@@ -155,6 +156,8 @@ NOTE: It is an error to specify both `dictionary` and `dictionary_path`.
 
   * Value type is <<path,path>>
   * There is no default value for this setting.
+  * This setting _cannot_ be combined with `dictionary`
+  * See also: <<plugins-{type}s-{plugin}-refresh_interval>> and <<plugins-{type}s-{plugin}-refresh_behaviour>>
 
 The full path of the external dictionary file. The format of the table should be
 a standard YAML, JSON, or CSV. 
```

Comment:
From the initial request for clarification we also need to make clear that the contents are entirely stored in memory. A suggestion:

```suggestion
  * See also: <<plugins-{type}s-{plugin}-refresh_interval>> and <<plugins-{type}s-{plugin}-refresh_behaviour>>

The full path of the external dictionary file.
The dictionary is read into memory to allow fast lookups, so make sure the JVM Heap is sized to fit its entire content.
The format of the table should be a standard YAML, JSON, or CSV. 
```